__tests__/app.test.js

```diff
@@ -96,6 +96,7 @@ describe("GET /api/reviews/:review_id", () => {
         expect(review).toHaveProperty("votes", 5);
         expect(review).toHaveProperty("created_at", "2021-01-18T10:01:41.251Z");
         expect(Date.parse(review.created_at)).toBe(1610964101251);
+        expect(review).toHaveProperty("comment_count", 3);
```

Comment:
This shouldnt be added to the existing test you have, but instead should be a new test.